### PR TITLE
CSS for days!

### PR DESCRIPTION
**Pull request by RazvanBudacaTechquarter**
Mostly stylizing the pages.

Each page has been given some kind of style and the band generator got more moods and bands to choose from. I'm pretty sure CSS stands for Can't Stop Swearing.

### CODE REVIEW

File: portfolio/src/main/webapp/bandGen.js [hunk 1]
```diff
@@ -5,20 +5,28 @@ function getMood() {
 }
 
 function generateBand() {
-    const NUMBER_OF_BANDS = 2;
+    const NUMBER_OF_BANDS = 5;
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/bandGen.js]: nit: From the [style guide](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names): "Constants in function scopes are still named in `lowerCamelCase`"

File: portfolio/src/main/webapp/playingInGCG.html [hunk 2]
```diff
@@ -3,18 +3,23 @@
...
     <meta charset="UTF-8">
     <title>Playing in GCG</title>
-    <link rel="stylesheet" href="styleGCG.css">
+    <link rel="stylesheet" href="style/styleGCG.css">
   </head>
   <body>
-    <div id="intro">
       <h1>Playing in GCG</h1>
+      <div id="content">
+      <div id="intro">
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/playingInGCG.html]: Fix the indentation, so it's easier to read that "intro" is inside "content"

File: portfolio/src/main/webapp/style/style.css [hunk 3]
```diff
@@ -24,20 +24,32 @@ a:hover, a:active {
...
 ul {
-    display: flex;
-    align-content: flex-end;
-    margin-right: 50px;
+  display: flex;
+  align-content: flex-end;
+  margin-right: 50px;
 }
 
 #picDesc {
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style/style.css]: nit: the [style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters) suggests using hyphen to separate IDs, so "pic-desc" here and "band-container" below

File: portfolio/src/main/webapp/style/styleBooks.css [hunk 4]
```diff
@@ -5,4 +5,17 @@
...
+  width: 100%;
+  background-color: #34623F;
+  color: white;
+  font-size: 70px;
+}
+
+img {
+    width: 100%;
+    height: 80%;
+
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style/styleBooks.css]: nit: remove extra line

File: portfolio/src/main/webapp/style/styleGCG.css [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+h1 {
+  width: 100%;
+  background-color: #34623F;
+  color: white;
+  font-size: 70px;
 }
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style/styleGCG.css]: This seems to be the same across every file. In this situation, I suggest moving it to `style.css`. The files which you want this style applied to can import multiple CSS files, e.g. `styleGCG.css` and `style.css`. 

Same thing for other elements that you might be sharing the style.

File: portfolio/src/main/webapp/bandGen.js [hunk 6]
```diff
@@ -5,20 +5,28 @@ function getMood() {
...
 }
 
 function getBands() {
     let bands = {};
-    bands["Chill"] = ["Elijah Nang", "Joji"];
-    bands["Funky"] = ["Marc Rebillet", "TWRP"];
-    bands["Ecstatic"] = ["Big Bad Voodoo Daddy", "Taraf de Haidouks"];
-    bands["Fancy"] = ["Devil Doll", "The Speakeasies Swing Band"];
+    bands["Chill"] = ["Elijah Nang", "Joji", "Syros", "DopeSmoker",
+    "Mystic Sons"];
```
Comment (jgaillard85) [inline portfolio/src/main/webapp/bandGen.js]: Indent this line more to make it clear that it is the continuation of the previous line.
Same for all the other line breaks that follow

File: portfolio/src/main/webapp/style/styleRecording.css [hunk 7]
```diff
@@ -1,3 +1,26 @@
 .hidden {
-    display: none;
+  display: none;
+}
+
+#hiddenContent {
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style/styleRecording.css]: hidden-content, per style guide

File: portfolio/src/main/webapp/style/styleRecording.css [hunk 8]
```diff
@@ -1,3 +1,26 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+
```
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style/styleRecording.css]: nit: remove extra line

**Review by amaurymedeiros — APPROVED** [review]
Please address the minor comments before merging the pull request

**Pull request merged**